### PR TITLE
wallet-ext: new ui for stake landing view

### PR DESCRIPTION
**Pull request by pchrysochoidis**
* make logo clickable - navigate to tokens view
* show/hide bottom navigation based on location path
* adds new stake landing page - only enabled in dev mode - uses mock data

https://user-images.githubusercontent.com/10210143/180938201-bdab0ce7-11f5-4075-851f-aaf6e0cf943c.mov

part of #3053

### CODE REVIEW

**Review by Jibz1 — APPROVED** [review]

**Review by gegaowp — APPROVED** [review]

**Review by Clay-Mysten — APPROVED** [review]
The small markdown changes LGTM here. I do wonder whether we should update the wallet docs with this functionality though:
https://docs.sui.io/explore/wallet-browser

File: wallet/font-icons/README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ It will be useful to also document here the source of each icon.
 -   [coins.svg](./svgs/coins.svg) - exported from [figma](https://www.figma.com/file/OzLaRFzevjxdQAbybWEZk0/Sui-Visualize?node-id=2251%3A47447)
 -   [hand-coins.svg](./svgs/hand-coins.svg) - exported from [figma](https://www.figma.com/file/OzLaRFzevjxdQAbybWEZk0/Sui-Visualize?node-id=2251%3A47447)
 -   [percentage-polygon.svg](./svgs/percentage-polygon.svg) - exported from [figma](https://www.figma.com/file/OzLaRFzevjxdQAbybWEZk0/Sui-Visualize?node-id=2251%3A47447)
+-   [close.svg](./svgs/close.svg) - exported from [figma](https://www.figma.com/file/rkFrheddol8YO7HQaHgIfF/Sui-Systematize?node-id=3421%3A2392)
+-   [arrow-right.svg](./svgs/arrow-right.svg) - exported from [figma](https://www.figma.com/file/rkFrheddol8YO7HQaHgIfF/Sui-Systematize?node-id=3421%3A2392)
```
Comment (randall-Mysten) [inline wallet/font-icons/README.md:27]: wondering whether these should be Figma since referring to a formal product.

**Review by randall-Mysten — APPROVED** [review]

**Comment by pchrysochoidis** [comment]
> The small markdown changes LGTM here. I do wonder whether we should update the wallet docs with this functionality though: https://docs.sui.io/explore/wallet-browser

If you mean the font-icons thing, we could but I am not sure if it's necessary. This is only required if someone wants to update the font-icons. It's not required for example for building the app etc.

If you mean the new view probably we should but when it's ready. Now it's only available in dev mode

File: wallet/font-icons/README.md (hunk 1 above)
Comment (pchrysochoidis) [reply wallet/font-icons/README.md:27]: figma -> Figma ?

or remove link?

**Pull request merged**